File: app/src/main/java/com/ceandt/moviedb/model/Genre.java
package com.ceandt.moviedb.model;

import com.google.gson.annotations.SerializedName;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Genre {

    @SerializedName("id")
    private Integer id;
    @SerializedName("name")
    private String name;

}

File: app/src/main/java/com/brunodiegom/movies/model/Genre.kt
package com.brunodiegom.movies.model

import com.google.gson.annotations.SerializedName

/**
 * Data object to parse the [Genre] data.
 */
data class Genre(
    @SerializedName("id")
    val id: Int,
    @SerializedName("name")
    val name: String
)
